### DISCUSSION TITLE
How can I insert image with srcset in markdown?

### DISCUSSION THREAD

**Discussion by lonhtm** [sitepress/sitepress#53] [Q&A]
I would like to insert assets (from rails assets) directly into markdown file form srcset attribute like 

```html
<figure><img srcset="/images/guides/pricing.png 1x, /images/guides/pricing@2x.png 2x" alt="Subcription plans"/></figure>
```

**Comment by MSchmidt** [comment]
You can override the image markdown element. Start with the guide: https://sitepress.cc/basics/markdown

Then add the following to your `ApplicationMarkdown`
```ruby
def image(link, title, alt)
  image_tag link, title: title, alt: alt
end
```

So far it's the same behaviour as before but now you can add your custom figure code there.
